x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -399,12 +396,14 @@ export class SessionIndex {
     }
 
     try {
-      const response = await this.options.clusterClient.callAsInternalUser('deleteByQuery', {
-        index: this.indexName,
-        refresh: 'wait_for',
-        ignore: [409, 404],
-        body: { query: { bool: { should: deleteQueries } } },
-      });
+      const { body: response } = await this.options.elasticsearchClient.deleteByQuery(
+        {
+          index: this.indexName,
+          refresh: true,
```

Comment:
__note:__ `wait_for` is actually not supported by the [`deleteByQuery`](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#_refreshing_shards)